src/diffusers/pipelines/stable_diffusion_xl/pipeline_flax_stable_diffusion_xl_img2img.py

```diff
@@ -0,0 +1,396 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
@pcuenca does this work already? Otherwise we could also move it out of this PR and do it in a follow up PR as it shouldn't really block anything for the demo